code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -246,6 +261,11 @@ var/global/mulebot_count = 0
 			auto_pickup = !auto_pickup
 		if("report")
 			report_delivery = !report_delivery
+		if("ejectpai")
+			if(bot_core.allowed(user) && paicard)
+				speak("Ejecting personality chip.", radio_channel)
+				ejectpai(user)
+			return

```

Comment:
This is copy-pasted from bot_control
Please make it a proc and call it on both instead of using copy-paste